src/lsdb/core/crossmatch/kdtree_match.py

```diff
@@ -18,7 +18,7 @@ class KdTreeCrossmatch(AbstractCrossmatchAlgorithm):
     def crossmatch(
         self,
         n_neighbors: int = 1,
-        d_thresh: float = 0.01,
+        d_thresh: float = 0.000278,  # 1 arcsecond
```

Comment:
Would it make sense to change the units on this input? If the values are generally going to be less than a degree, then it might be silly, but if astronomers expect degrees then it's probably ok. Really, I'm asking @hombit or @nevencaplar though.

Reply:
1. I would write `1.0 / 3600.0` instead here (if we keep degrees)
2. There is no single value people use across the APIs and libraries, I've seen all possible units: degrees, arcminutes, arcseconds, and even radians. I would prefer to use arcseconds, because it the typical scale we are aimed to support
3. I would rename the argument to something like `radius_arcsec` to make the units more explicit for users

Reply:
I support arcseconds. @delucchi-cmu Are there many place where we would need to change API? If yes, does that deserve another ticket?

Reply:
Does everyone agree on the `radius_arcsec` name change? Should we have more explicit units in names everywhere? The only other radius I can think of right now is the cone search radius. Does that make more sense in arcsec too?